src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/Main.cs

```diff
@@ -110,7 +130,34 @@ public List<Result> Query(Query query)
             }
             catch (FormatException e)
             {
-               Log.Debug(GetTranslatedPluginTitle() + ": " + e.Message, GetType());
+                Log.Debug(GetTranslatedPluginTitle() + ": " + e.Message, GetType());
+                if (!string.IsNullOrEmpty(query.ActionKeyword))
+                {
+                    return GetSuggestionFuzzyResults(query, results);
+                }
+            }
+
+            return results;
+        }
+
+        private List<Result> GetSuggestionResults(Query query, List<Result> results)
+        {
+            foreach (var generatorData in QueryHelper.GeneratorDataList)
+            {
+                results.Add(new Result
+                {
+                    Title = QueryHelper.GetResultTitle(generatorData),
+                    SubTitle = QueryHelper.GetResultSubtitle(generatorData),
+                    IcoPath = GetIcoPath(),
+                    ToolTipData = new ToolTipData(QueryHelper.GetResultTitle(generatorData), QueryHelper.GetResultSubtitle(generatorData)),
+                    ToolTipVisibility = Visibility.Visible,
+                    QueryTextDisplay = generatorData.Keyword,
```

Comment:
Don't we need both: Action keyword and generator keyword?

Reply:
![image](https://github.com/microsoft/PowerToys/assets/24465401/11450d0c-68d2-44f8-ad19-b741597d5cfc)
No, this is what will happen if both is included

Reply:
💡